frontend/spec/features/checkout_spec.rb

```diff
@@ -207,16 +188,14 @@
       visit spree.checkout_state_path(:payment)
     end
 
-    it 'the first payment method should be selected' do
-      payment_method_id_prefix = 'order_payments_attributes__payment_method_id_'
-      expect(page).to have_checked_field(id: "#{payment_method_id_prefix}#{check_payment.id}")
-      expect(page).to have_unchecked_field(id: "#{payment_method_id_prefix}#{credit_cart_payment.id}")
+    it 'the credit card payment method should be visible' do
+      expect(page).to have_css("#payment_method_#{credit_card_payment.id}")
```

Comment:
Should we add `visible: true` here?

Reply:
From what I know this is not necessary as `visible: true` is the default option when checking `css`